### DISCUSSION TITLE
Unable to comment in the issues list...

### DISCUSSION THREAD

**Discussion by wcndave** [chocolatey/choco#3584] [Q&A]
I cannot write a comment to respond to another comment about an issue I raised... really one of **those** days....

![image](https://github.com/user-attachments/assets/8fb73cf6-968c-4ac2-9ce8-7fd068ac901e)

**Comment by pauby** [comment]
This is a GitHub issue. Retrying later usually works. If it doesn't, please contact GitHub support.
